src/components/ParameterTypeRegistrationModal/index.tsx

```diff
@@ -64,7 +73,7 @@ const ParameterRegistrationModal = forwardRef<ParameterTypeRegistrationModalRef,
 							/>
 
 							<Label>Unidade de medida</Label>
-							<Select onChange={(event) => setUnitParameter(event.target.value)}>
+							<Select onChange={(event: any) => setUnitParameter(event.target.value)}>
```

Comment:
`any` 😭

Reply:
Vou ajustar isso na branch [feat/add-parameters](https://github.com/The-Bugger-Ducks/cloud-fox-web/tree/feat/add-parameters) pois estou refazendo esse modal nela, pode ser?

Reply:
Quando for adicionar é essa a tipagem do select 

```ts
(event: React.ChangeEvent<HTMLSelectElement>)
```